ci/kind/test-e2e-kind.sh

```diff
@@ -235,9 +235,11 @@ done
 # The Antrea images should not be pulled, as we want to use the local build.
 if $coverage; then
     manifest_args="$manifest_args --coverage"
-    COMMON_IMAGES_LIST+=("antrea/antrea-ubuntu-coverage:latest")
+    COMMON_IMAGES_LIST+=("antrea/antrea-agent-ubuntu-coverage:latest" \
+                         "antrea/antrea-controller-ubuntu-coverage:latest")
 else
-    COMMON_IMAGES_LIST+=("antrea/antrea-ubuntu:latest")
```

Comment:
I am afraid you need to check which scripts are using this image antrea/antrea-ubuntu:latest and update them accordingly considering the antrea.yml is updated to use two new images.  @XinShuYang  could you help to review this PR as well. Thanks.

Reply:
We also need to update the Jenkins script for image building and delivery, which will have an impact on most online jobs. https://github.com/antrea-io/antrea/blob/792e2447ae4ed0ad6f6293f4b1a39cf98e936e5b/ci/jenkins/test.sh#L415 https://github.com/antrea-io/antrea/blob/792e2447ae4ed0ad6f6293f4b1a39cf98e936e5b/ci/jenkins/test.sh#L609

Reply:
@luolanzone For this patch, I feel we should trigger most Jenkins jobs following the standards of the release PR.